admin/services/data-access/group.data.service.js

```diff
@@ -2,8 +2,13 @@
 
 const Group = require('../../models/group')
 const groupDataService = {}
+const sqlService = require('./sql.service')
+const TYPES = require('tedious').TYPES
+const moment = require('moment')
+const R = require('ramda')
 
 /**
+ * @TODO: To be deleted when the data layer refactoring is done.
```

Comment:
please add `@deprecated` with a message informing what the replacment sql function is called.